Tools/CMake/AMReX_Config.cmake

```diff
@@ -155,7 +155,7 @@ function (configure_amrex)
         if( host_link_supported STREQUAL "NEW" )
           target_link_options(amrex PUBLIC "$<HOST_LINK:LINKER:--warn-unresolved-symbols>")
         else()
-          target_link_options(amrex PUBLIC "LINKER:--warn-unresolved-symbols")
+          target_link_options(amrex PUBLIC "-Wl,--warn-unresolved-symbols")
```

Comment:
@robertmaynard in line 144: should we roll back or make the `LINKER:` conditional as well?
We don't have CUDA these days on macOS anymore, but if `LINKER:` is a CMake 3.18+ feature we might better not place it in the `if(APPLE)` path unconditionally, yet.